SPECS/libacvp/libacvp.spec

```diff
@@ -35,7 +36,7 @@ and the library itself.
 ./configure \
     --prefix=%{_prefix} \
     --enable-offline \
-    CFLAGS="-pthread -DACVP_NO_RUNTIME -DOPENSSL_KWP -DOPENSSL_KDF_SUPPORT -O0 -g -fcommon" \
+    CFLAGS="-pthread -DACVP_NO_RUNTIME -DOPENSSL_KWP -DOPENSSL_KDF_SUPPORT -O2 -g -fcommon" \
```

Comment:
Curious: why was this necessary?

Reply:
`_FORTIFY_SOURCE` has no effect with `-O0` (also why build a non-optimized library in a package)